plugins/tiddlywiki/codemirror-autocomplete/files/tiddlywiki.files

```diff
@@ -5,35 +5,40 @@
 			"fields": {
 				"type": "application/javascript",
 				"title": "$:/plugins/tiddlywiki/codemirror/addon/hint/anyword-hint.js",
-				"module-type": "codemirror"
+				"module-type": "codemirror",
+				"require-index": "18"
```

Comment:
hmmm, we should find a way to do this programmatically. ... Doing it be hand is a lot of work and users can't add their own stuff without problems.